admin/app/com/lucidchart/piezo/admin/controllers/HealthCheck.scala

```diff
@@ -0,0 +1,39 @@
+package com.lucidchart.piezo.admin.controllers
+
+import org.joda.time.{DateTime,Minutes}
+import org.joda.time.format.ISODateTimeFormat
+import play.api._
+import play.api.Play.{current,configuration}
+import play.api.mvc._
+import scala.io.Source
+
+object HealthCheck extends Controller {
+  implicit val logger = Logger(this.getClass())
+
+  val heartbeatFilename = configuration.getString("com.lucidchart.piezo.heartbeatFile").getOrElse {
+    logger.warn("heartbeat file not specified")
+    ""
+  }
+  val minutesBetweenBeats = configuration.getInt("healthCheck.worker.minutesBetween").getOrElse{
+    logger.warn("minutes between heartbeats not specified. Defaulting to 5")
+    5
+  }
+
+  def main = Action { implicit requests =>
+   if(workersAreHealthy) {
+     Ok
```

Comment:
Will you return the last heartbeat timestamp here (in both health and unhealthy actually)?